etc/example-config.yaml

```diff
@@ -35,3 +35,5 @@ broker:
   launch_apb_on_bind: false
   recovery: true
   output_request: true
+  ssl_cert_key: /path/to/key
```

Comment:
what does this have to do with the dev_broker? I would rather never show this as being able to accept an empty string as that could be confusing.